### PR TITLE
*: update siddontang/go-mysql to go-mysql-org/go-mysql

### PR DESCRIPTION
**Pull request by lance6716**
### What problem does this PR solve? 

upstream siddontang/go-mysql has been renamed to go-mysql-org/go-mysql

### What is changed and how it works?

- update siddontang/go-mysql to go-mysql-org/go-mysql
- add a unit test 

### Check List 

Tests 

 - Unit test

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

File: pkg/gtid/gtid_test.go [hunk 1]
```diff
@@ -85,12 +85,15 @@ func (s *testGTIDSuite) TestGTID(c *C) {
...
+	gSet, err := ParserGTID(mysql.MySQLFlavor, "3ccc475b-2343-11e7-be21-6c0b84d59f30:1-14,406a3f61-690d-11e7-87c5-6c92bf46f384:1-94321383,53bfca22-690d-11e7-8a62-18ded7a37b78:1-495,05474d3c-28c7-11e7-8352-203db246dd3d:1-170,10b039fc-c843-11e7-8f6a-1866daf8d810:1-308290454,686e1ab6-c47e-11e7-a42c-6c92bf46f384:1-34981190,03fc0263-28c7-11e7-a653-6c0b84d59f30:1-7041423")
 	c.Assert(err, IsNil)
 	sortedGTIDSet := "03fc0263-28c7-11e7-a653-6c0b84d59f30:1-7041423,05474d3c-28c7-11e7-8352-203db246dd3d:1-170,10b039fc-c843-11e7-8f6a-1866daf8d810:1-308290454,3ccc475b-2343-11e7-be21-6c0b84d59f30:1-14,406a3f61-690d-11e7-87c5-6c92bf46f384:1-94321383,53bfca22-690d-11e7-8a62-18ded7a37b78:1-495,686e1ab6-c47e-11e7-a42c-6c92bf46f384:1-34981190"
-	c.Assert(sortedGTIDSet, Equals, gSet.String())
+	c.Assert(gSet.String(), Equals, sortedGTIDSet)
 	// check mariadb
-	// TODO: when go-mysql add sorting for mariadb, finish here
+	gSet, err = ParserGTID(mysql.MariaDBFlavor, "0-0-1,1-1-1,4-20-1,3-1-1,10-10-10")
```
Comment (lance6716) [inline pkg/gtid/gtid_test.go:93]: this is the added unit test

**Review by Ehco1996 — APPROVED** [review]

**Pull request merged**